src/js/audit-submissions.js

```diff
@@ -1,18 +1,48 @@
 (function () {
   function init() {
+    const START_SUBMISSION_URL = '../audit/new/step-1/';
```

Comment:
Can this `init` function be broken up a little bit? It's really big. Splitting it into some smaller functions might help readability/maintainability.